apps/blestress/src/stress.h

```diff
@@ -39,7 +39,7 @@ extern "C" {
 #define TX_PHY_MASK 0
 #define RX_PHY_MASK 0
 /* L2CAP SDU */
-#define STRESS_COC_MTU (64000)
```

Comment:
why we need to decrease it ?

Reply:
I'm not sure what was wrong there, reading commit message I mentioned some issue with `ble_l2cap_coc_rx_fn`. Whatever this was, was fixed and this commit is not needed - I retested it. I suspect that https://github.com/apache/mynewt-nimble/commit/c1a2c9949805148520afb852d86afe588061d64e was the fix